invenio_records_lom/cli.py

```diff
@@ -33,6 +34,32 @@ def rebuild_index():
     click.secho("Reindexed records!", fg="green")
 
 
+@lom.command()
+@with_appcontext
+def check():
+    """Check records in SQL-database against marshmallow-schema for OAI.
+
+    Note: this does not guarantee by itself that OAI-PMH API works correctly, since
+    (1) Records in opensearch might not mirror records in SQL-database, call `invenio lom reindex` to remedy this.
+    (2) Records that pass OAI-schema verification might still not play nicely with OER-Hub
```

Comment:
don't mention OERhub directly. use the general word `harvesters`.

Reply:
done